app/database/group_store.go

```diff
@@ -14,8 +14,11 @@ func (s *GroupStore) ManagedBy(user *User) *DB {
 				ON groups_ancestors_active.child_group_id = groups.id`).
 		Joins(`
 			JOIN group_managers
-				ON group_managers.group_id = groups_ancestors_active.ancestor_group_id AND
-					group_managers.manager_id = ?`, user.GroupID)
+				ON group_managers.group_id = groups_ancestors_active.ancestor_group_id`).
```

Comment:
Why not just using `func (s *GroupAncestorStore) ManagedByUser(user *User)`  defined above?

Reply:
ManagedByUser() is a method of GroupAncestorStore which we don't have here. We cannot call this method on the result of 'Joins' and we cannot convert the store into GroupAncestorStore either as it would change the main table name.